doc/introduction/measurements.rst

```diff
@@ -37,3 +35,76 @@ The three measurement functions can be found here:
 
 :html:`</div>`
 
+.. note::
+
+    Both :func:`~.pennylane.expval` and :func:`~.pennylane.var` support analytic
+    differentiation. :func:`~.pennylane.sample`, however, returns *stochastic*
+    results, and does not support differentiation.
+
+Multiple measurements
+---------------------
+
+Quantum functions can also return multiple measurements, as long as each wire
+is not measured more than once:
+
+.. code-block:: python
+
+    def my_quantum_function(x, y):
+        qml.RZ(x, wires=0)
+        qml.CNOT(wires=[0, 1])
+        qml.RY(y, wires=1)
+        return qml.expval(qml.PauliZ(1)), qml.var(qml.PauliX(0))
+
+You can also use list comprehensions, and other common Python patterns:
+
+.. code-block:: python
+
+    def my_quantum_function(x, y):
+        qml.RZ(x, wires=0)
+        qml.CNOT(wires=[0, 1])
+        qml.RY(y, wires=1)
+        return [qml.expval(qml.PauliZ(i)) for i in range(2)]
+
+Tensor observables
+------------------
+
+PennyLane supports measuring the tensor product of observables, by using
+the ``@`` notation. For example, to measure the expectation value of
+:math:`Z\otimes I \otimes X`:
+
+.. code-block:: python3
+
+    def my_quantum_function(x, y):
+        qml.RZ(x, wires=0)
+        qml.CNOT(wires=[0, 1])
+        qml.RY(y, wires=1)
+        qml.CNOT(wires=[0, 2])
+        return qml.expval(qml.PauliZ(0) @ qml.PauliX(2))
```

Comment:
maybe be good to note that there's no need to declare the identity term?

Reply:
I thought that might be confusing/overcomplicate - thoughts?

Reply:
done